agent/http.go

```diff
@@ -368,6 +368,9 @@ func (s *HTTPHandlers) wrap(handler endpoint, methods []string) http.HandlerFunc
 				}
 				logURL = strings.Replace(logURL, token, "<hidden>", -1)
 			}
+			httpLogger.Warn("This request is using a token query parameter "+
```

Comment:
There's only one, so we should say "the" token query param. "a" suggests there might be multiple query parameters that could include a token, rather than that the query param is named literally "token". (At least by my read)

```suggestion
			httpLogger.Warn("This request is using the token query parameter "+
```